### PR TITLE
set digit length to 10

### PR DESCRIPTION
**Pull request by arsh1598**

### CODE REVIEW

**Comment by dsyer** [comment]
Doesn't that duplicate the existing `@Digits` annotation? The intent seems to be the same at least.

**Comment by arsh1598** [comment]
> Doesn't that duplicate the existing `@Digits` annotation? The intent seems to be the same at least.

@digits sets maximum digit limit not minimum.

File: src/main/java/org/springframework/samples/petclinic/owner/Owner.java [hunk 1]
```diff
@@ -56,7 +56,7 @@ public class Owner extends Person {
 
 	@Column(name = "telephone")
 	@NotEmpty
```
Comment (andrzejsydor) [inline src/main/java/org/springframework/samples/petclinic/owner/Owner.java:58]: so the not empty is probably redundant?

Comment (dsyer) [reply src/main/java/org/springframework/samples/petclinic/owner/Owner.java:58]: Don't know. What would be the effect of removing it? None of these errors has any error messages or nice handling associated with them anyway, so it's all kind of moot.

**Comment by arsh1598** [comment]
> I see. But not all telephone numbers have 10 digits, so `@Digits` seems like the right tool to me still. It's not really a big deal.

closing this PR

**Pull request closed without merging**